tests/unit/test_json.py

```diff
@@ -0,0 +1,16 @@
+import json
```

Comment:
Nice!
Can we also add a design doc, on what we hope to accomplish with this test framework?

Reply:
Open to creating a design doc section, sure. Will discuss offline.